pulse/inngest-router/README.md

```diff
@@ -0,0 +1,130 @@
+# Prisma Pulse-Inngest Router
+
+This package enables event-driven support for your existing Inngest workflows,
```

Comment:
for `.md` files we generally don't line break like this. I'm assuming this is a Prettier thing? Would it be possible to edit the Prettier configuration to ignore `.md` files?

Reply:
No worries, fixed. It's an old habit of breaking lines at 80 characters (totally manually 😅)